libbeat/common/datetime_test.go

```diff
@@ -41,7 +43,7 @@ func TestParseTime(t *testing.T) {
 		{
 			Input:  "2015-04-31T11:19:05.112Z",
 			Output: time.Date(2015, time.April, 31, 11, 19, 05, 112*1e6, time.UTC),
-		},
+		},*/
```

Comment:
problem with golang 1.6. They were now removed anyways in https://github.com/elastic/beats/pull/1042, hence the required rebase.